docs/developer-docs/gas-cost.md

```diff
@@ -24,9 +24,9 @@ Canisters pay for consumed resources by burning cycles. These resources include
 
 ## Units and fiat value
 
-The price of cycles is fixed against the price of [XDR](/docs/current/references/glossary#xdr), where **1 trillion cycles equals 1 XDR**. As of December 18, 2023, the exchange rate for 1 XDR = $1.336610, which is used on this page. The exchange rate for USD/XDR may vary, and it will impact the conversion rate. You can view XDR exchange rates [here](https://www.imf.org/external/np/fin/data/rms_sdrv.aspx).
+The price of cycles is fixed against the price of [XDR](/docs/current/references/glossary#xdr), where **1 trillion cycles equals 1 XDR**. As of December 18, 2023, the exchange rate for 1 XDR = $1.336610. The exchange rate for USD/XDR may vary. Learn more about [XDR exchange rates](https://www.imf.org/external/np/fin/data/rms_sdrv.aspx).
```

Comment:
I propose to shorten sentences with links by not separating the description of where the visitor might want to go and the link (see also my comment on Slack):

Instead of "You can view XDR exchange rates [here](https://www.imf.org/external/np/fin/data/rms_sdrv.aspx)"
Write "More [XDR exchange rates](https://github.com/dfinity/portal/pull/2297/link)".

Reply:
Revised to " Learn more about [XDR exchange rates](link)"